pynecone/components/media/image.py

```diff
@@ -50,3 +52,11 @@ def get_triggers(self) -> Set[str]:
             The event triggers.
         """
         return super().get_triggers() | {"on_error", "on_load"}
+
+    def _render(self) -> Tag:
+        # If given a pandas df break up the data and columns
+        if types.is_image(type(self.src)):
+            self.src = Var.create(format.formant_image_data(self.src))  # type: ignore
```

Comment:
I think we can remove the `type: ignore` after changing to `Var[Any]`

Reply:
We need it here